tailwind.config.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
         500: "#393a60",
         600: "#2e2e4d",
         700: "#22233a",
-        800: "#171726",
+        500: "#171726",
```

Comment:
Was this change intentional?

Reply:
Oh no! Good catch! I will revert this!

Reply:
@dnsos we never use the gray colors in this website. Should we keep them in the config?

Reply:
Good point. Feel free to remove them with this PR or separately.